src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_InvalidExpression.cs

```diff
@@ -570,5 +570,139 @@ void F(int p /*<bind>*/= M()/*</bind>*/)
 
             VerifyOperationTreeAndDiagnosticsForTest<EqualsValueClauseSyntax>(source, expectedOperationTree, expectedDiagnostics);
         }
+
+        [Fact]
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [WorkItem(20050, "https://github.com/dotnet/roslyn/issues/20050")]
+        public void BuildsArgumentsOperationsForDuplicateExplicitArguments_Repro()
+        {
+            string source = @"
+public class C
+{
+    void M()
+    {
+        /*<bind>*/string.Format(format: """", format: """")/*</bind>*/;
+    }
+}";
+            string expectedOperationTree = @"
+IInvocationOperation (System.String System.String.Format(System.String format, params System.Object[] args)) (OperationKind.Invocation, Type: System.String, IsInvalid) (Syntax: 'string.Form ... format: """")')
+  Instance Receiver: 
+    null
+  Arguments(3):
+      IArgumentOperation (ArgumentKind.Explicit, Matching Parameter: format) (OperationKind.Argument, Type: null) (Syntax: 'format: """"')
+        ILiteralOperation (OperationKind.Literal, Type: System.String, Constant: """") (Syntax: '""""')
+        InConversion: CommonConversion (Exists: True, IsIdentity: True, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
+        OutConversion: CommonConversion (Exists: True, IsIdentity: True, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
+      IArgumentOperation (ArgumentKind.Explicit, Matching Parameter: format) (OperationKind.Argument, Type: null, IsInvalid) (Syntax: 'format: """"')
```

Comment:
>IArgumentOperation (ArgumentKind.Explicit, Matching Parameter: format) (OperationKind.Argument, Type: null, IsInvalid) (Syntax: 'format: """"') [](start = 6, length = 143)

I think this is wrong, we are not supposed to create IInvocationOperation when there are issues with mapping arguments to parameters. The IInvocationOperation can be created only for successful overload resolution case.